graphql/context.go

```diff
@@ -17,6 +17,10 @@ type RequestContext struct {
 	RawQuery  string
 	Variables map[string]interface{}
 	Doc       *ast.QueryDocument
+
+	ComplexityLimit     int
```

Comment:
why is ComplexityLimit here, its static?

Reply:
Yes it's static value determined at compile time.

BTW, I want to modify it on run time...

Reply:
> BTW, I want to modify it on run time...

:thinking: 

 when? ideally a request over the limit doesn't run anything, especially not in the resolver stack.

Reply:
For example, If we want to change ComplexityLimit between 1st party client and 3rd party client.